osu.Game/Graphics/Containers/OsuScrollContainer.cs

```diff
@@ -0,0 +1,75 @@
+﻿// Copyright (c) 2007-2017 ppy Pty Ltd <contact@ppy.sh>.
+// Licensed under the MIT Licence - https://raw.githubusercontent.com/ppy/osu/master/LICENCE
+
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Input;
+using OpenTK.Input;
+
+namespace osu.Game.Graphics.Containers
+{
+    internal class OsuScrollContainer : ScrollContainer
+    {
+        /// <summary>
+        /// Add the ability to seek to an absolute scroll position when the right mouse button is pressed or dragged.
+        /// Uses the value of <see cref="DistanceDecayOnRightMouseScrollbar"/> to smoothly scroll to the dragged location.
+        /// </summary>
+        public bool RightMouseScrollbar = false;

```

Comment:
i decided not to use "relative" because as discussed with @Tom94, depending on how you think about it it could be absolute *or* relative.

personally i think of it as "absolute" scroll since you are scrolling to an absolute position, unlike normal drags where you are scrolling relative to the current y offset.